### DISCUSSION TITLE
Amiberry on Steam Deck???

### DISCUSSION THREAD

**Discussion by Panthros** [BlitterStudio/amiberry#1146] [Q&A] [answered]
Anyone able to share how to get Amiberry working with Archlinux on Steam Deck?  There was talk of a Flatpak which would be awesome!

**Comment by midwan** [comment, marked as answer]
It's possible with a flatpak package, which I'm working on. The current version I'm testing has some issue, as it crashes right after it opens the GUI (previous versions worked, so I broke something). Once this is ready, the plan is to release flatpak binaries alongside a release. Ideally with the packages also uploaded to something like flathub, so you can get automatic updates or even install it from there directly.

The alternative is rather messy and painful, and involves a lot of manual tinkering to allow package installations and updates on the filesystem itself, then compiling the binary from source, in order to run it. And any new firmware updates will wipe out your changes, so you'd have to do it again.

So flatpak is the only reasonable way forward for this. :)

**Reply by midwan** [reply]
Note: the latest builds fixed the crash I had, so it works as expected on the SteamDeck now.

**Reply by Juppstein** [reply]
Just as a FYI I tried the Flatpak on Ubuntu 22.04 and it immediately crashes there.

**Reply by midwan** [reply]
I haven't tried it on Ubuntu, but I've tested it on Arch and Manjaro on three different cases here, and it works normally. Will have to set up an Ubuntu image at some point to test, but probably after the next release, which is very close now (final testing).
